docs/resources/dbss_instance.md

```diff
@@ -0,0 +1,176 @@
+---
+subcategory: "Database Security Service (DBSS)"
+---
+
+# huaweicloud_dbss_instance
+
+Manages a DBSS instance resource within HuaweiCloud.
+
+## Example Usage
+
+```HCL
+variable "name" {}
+variable "availability_zone" {}
+variable "vpc_id" {}
+variable "subnet_id" {}
+variable "security_group_id" {}
+
+resource "huaweicloud_dbss_instance" "test" {
+  name               = var.name
+  flavor             = "c3ne.xlarge.4"
+  availability_zone  = var.availability_zone
+  vpc_id             = var.vpc_id
+  subnet_id          = var.subnet_id
+  security_group_id  = var.security_group_id
+  charging_mode      = "prePaid"
+  period_unit        = "month"
+  period             = 1
+  product_id         = "00301-225396-0--0"
+  resource_spec_code = "dbss.bypassaudit.low"
+  product_spec_desc  = "test"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) The instance name. The name can contain 1 to 64 characters.
+  Only letters, digits, underscores (_), and hyphens (-) are allowed.
+
+  Changing this parameter will create a new resource.
+
+* `availability_zone` - (Required, String, ForceNew) The availability zone to which the instnce belongs.
+  Primary and secondary AZs are separated by commas. Example: cn-north-4a,cn-north-4b.
+
+  Changing this parameter will create a new resource.
+
+* `flavor` - (Required, String, ForceNew) Specifies the flavor.
+
+  Changing this parameter will create a new resource.
+
+* `vpc_id` - (Required, String, ForceNew) The VPC ID.
+
+  Changing this parameter will create a new resource.
+
+* `subnet_id` - (Required, String, ForceNew) The subnet ID of the NIC.
+
+  Changing this parameter will create a new resource.
+
+* `security_group_id` - (Required, String, ForceNew) Specifies the ID of the security group.
+
+  Changing this parameter will create a new resource.
+
+* `product_id` - (Required, String, ForceNew) The product ID.
```

Comment:
replaced with flavor_id?

Reply:
There is another parameter named `flavor`